### ISSUE TITLE
scrolling inside nested scrollview

### ISSUE THREAD

**Issue by Vasantika20** [google/flutter.widgets#199]
scrollable_positioned_list package with ScrollablePositionedList.builder i need it inside NestedScrollview, but if i use physics NeverScrollableScrollPhysics() then its not scrolling with the scroll of NestedScrollview.

I want to add headerSliverBuilder with NestedScrollview, is there any way to use header with ScrollablePositionedList? or any solution to use proper scroll inside NestedScrollview? 

Any help will be appreciated.
Thank you

**Comment by ayush-12** [comment]
I am facing the same issue. Did you found out any solution for this?

**Comment by ayush-12** [comment]
@tarobins  https://github.com/flutter/flutter/issues/12319#issuecomment-595435352 found this comment of you just wanted to check if there's any fix for scrollable_positioned_list to work with NestedScrollVIew

**Comment by Vasantika20** [comment]
No, i didn't found, instead of NestedScrollview i have used DraggableScrollableSheet.

**Comment by ayush-12** [comment]
@Vasantika20 Does it work with sliverAppBar and ScrollablePositionedList as expected?

**Comment by Vasantika20** [comment]
it will work as Google pay app, you can check in that. I am not using scrollable_positioned list, instead of that i have used RectGetter with CustomScrollView. You can check [https://gist.github.com/debuggerx01/49f108d68ed903458e9478b4f0c186f4](url)
